src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/AuthenticationTests.kt

```diff
@@ -0,0 +1,79 @@
+package org.vaccineimpact.orderlyweb.tests.integration_tests.tests
+
+import com.github.fge.jackson.JsonLoader
+import khttp.options
+import khttp.post
+import khttp.responses.Response
+import khttp.structures.authorization.Authorization
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.tests.integration_tests.helpers.RequestHelper
+import java.util.*
+
+class AuthenticationTests : IntegrationTest()
+{
+    @Test
+    fun `authentication fails without Auth header`()
+    {
+        val result = post(url)
+        assertThat(result.statusCode).isEqualTo(401)
+        JSONValidator.validateError(result.text,
+                expectedErrorCode = "github-token-invalid",
+                expectedErrorText = "GitHub token not supplied in Authorization header, or GitHub token was invalid")
+    }
+
+    @Test
+    fun `authentication fails with malformed Auth header`()
+    {
+        val result = post(url, auth = GithubTokenHeader("token","bearer"))
+        assertThat(result.statusCode).isEqualTo(401)
+        JSONValidator.validateError(result.text,
+                expectedErrorCode = "github-token-invalid",
+                expectedErrorText = "GitHub token not supplied in Authorization header, or GitHub token was invalid")
+    }
+
+    @Test
+    fun `authentication succeeds with well-formed Auth header`()
+    {
```

Comment:
At the moment because there's just dummy code instead of a real Github auth, any well-formed request gets logged in. It's going to be more challenging to test this once the Github auth code is in!